cfme/intelligence/reports/schedules.py

```diff
@@ -123,23 +129,20 @@ class Schedule(Updateable, Pretty, BaseEntity):
     Args:
         name: Schedule name.
         description: Schedule description.
-        filter: 3-tuple with filter selection (see the UI).
+        report_filter: 3-tuple with filter selection (see the UI).
         active: Whether is this schedule active.
-        run: Specifies how often this schedule runs. It can be either string "Once", or a tuple,
-            which maps to the two selects in UI ("Hourly", "Every hour")...
-        time_zone: Specify time zone.
-        start_date: Specify the start date.
-        start_time: Specify the start time either as a string ("0:15") or tuple ("0", "15")
-        send_email: If specifies, turns on e-mail sending. Can be string, or list or set.
+        timer: Specifies how often this schedule runs. It can be a dictionary that contains
+                run, run_interval, starting_date, timezone, starting_hour and starting_minute
+        send_email: If specified, turns on e-mail sending. Can be string, or list or set.
```

Comment:
Required:
@valaparthvi why doc-string not matching with actual arguments of class?